### DISCUSSION TITLE
CPU load 2x increase from pm2 agent?

### DISCUSSION THREAD

**Discussion by arminus** [SquirrelCorporation/SquirrelServersManager#38] [Q&A]
Another thing I noticed: I tested this in a Pi4 and also installed the agent on the same host. Seems that the PM2 agent doubles the CPU load from my nominal ~ 1.7 to 3.7 or above:

![2024-06-21_151540](https://github.com/SquirrelCorporation/SquirrelServersManager/assets/683680/f201cedd-d1ac-4d6e-811e-75a807257857)

Installed it around 11:15, shutdown the docker stack at around 13:00, uninstalled the agent around 15:00.

Is this kind of load to be expected/normal?

Side note: Uninstalling the agent from the UI still left parts behind (pm2-logrotate at least), had to uninstall that manually with `pm2 uninstall pm2-logrotate`

**Comment by SquirrelDeveloper** [comment]
Hi,
I did not experiment such cpu increased, but will take a look. I have added the cleaning of files to the todo

**Reply by Famku** [reply]
I also have no cpu increase problems on a Ubuntu VM

btw - it is really a cool app, looking forward for upcoming versions
